### PR TITLE
Infrastructure refactoring

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,7 +8,7 @@
 .vs/
 
 # Build results
-[Bb]inaries/
```
Comment (jmarolf) [inline .gitignore]: Why this change?

Comment (tmat) [inline .gitignore]: There is no Binaries directory, instead it's Artifacts.
